__DEFINES/setup.dm

```diff
@@ -871,6 +871,17 @@ SEE_PIXELS	256
 #define ROLE_VOXRAIDER  	"vox raider"
 #define ROLE_WIZARD     	"wizard"
 
+// Role typepaths.
+
+#define ROLE_CULTIST_TYPE /datum/role/cultist
```

Comment:
Why not use the type directly? what's the be gained from this?

Reply:
Not much, in retrospect.